src/CliUtils.js

```diff
@@ -19,8 +19,9 @@ function validateSteps(steps, sequencer) {
   // Assume all are valid in the beginning.
   var valid = true;
   steps.forEach(function(step) {
+    console.log(step.name);
```

Comment:
Ah, should this be removed? Thanks!

Reply:
Definitely.

Reply:
Yea sorry my bad :sweat_smile:

Reply:
```suggestion
```

I think this would be the last change, except for adding a test with a pipe character!